.husky/pre-commit

```diff
@@ -1,12 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
In husky v9, shebangs were deprecated: https://github.com/typicode/husky/releases/tag/v9.0.1

Reply:
This isn't a blocker though so I'm going to go ahead and approve. I can run through the test cases when they're ready!